### DISCUSSION TITLE
Add metadata to courses configuration file

### DISCUSSION THREAD

**Discussion by dimkard** [kantord/LibreLingo#2538] [General]
If I understand correctly, [this](https://github.com/LibreLingo/LibreLingo/blob/main/config/courses.json) file is manually edited, and it works as a configuration file that drives various CI/CD decisions.

On Glossaico, a similar configuration file is used which contains more or less the same information and:
- Display Name (e.g. Spanish)
- Description (e.g. Spanish for English speakers)

Do you think that such ^^ information can be added to courses.json or it is kind of "abuse" of the file with irrelevant information?

**Comment by kantord** [comment]
it coud be part of courses.json yes!
